src/utils/shopify.js

```diff
@@ -5,6 +5,7 @@ export async function shopifyFetch({ query, variables }) {
   const key =
     import.meta.env.VITE_SHOPIFY_STOREFRONT_API_TOKEN || 'ef7d41c7bf7e1c214074d0d3047bcd7b';
```

Comment:
It'd be better to use SvelteKit's env var handling here, as Vite's is very incomplete:

```js
import { SHOPIFY_STOREFRONT_API_TOKEN } from '$env/static/private';
```

The fallback value would need to be added to an `.env` file (or `.env.development` or whatever) that's checked into the repo.

Doing it this way means that the secret is only available to code that runs on the server — there's no way to accidentally leak secrets to the client, unlike `VITE_*`. (You also get autocomplete!) 

Aside: curious about the reason for the custom `$utils` alias rather than just putting this in `$lib`?